doc/tutorial.md

```diff
@@ -144,37 +153,74 @@ Estimated gas: Estimate (gas_used: 21, vm_status_code: EXECUTED)
 
 ## Finishing the Tutorial
 
 You have successfully published a Move module on your Polkadot blockchain and executed a Move script successfully.
 
-So now we are ready to buy some coins for washing our car using our Substrate balance. First, let's check the balance before we execute some new scripts! Open the menu __
-![Compare balances after Alice bought a coin from Bob](assets/polkadot.js_buy_coin_balances_before.png)
+| ![js_buy_coin_balances_after.png](assets/polkadot.js_buy_coin_balances_before.png) |
+|:--:|
+| _Observe the balance before we execute some new scripts which will charge the user_ |
 
 Now, let's execute the following actions:
 1. Using Alice's account, let's register her as a customer by executing the `register_new_user.mv` script
-  - Hint 1: this script requires one signer which has to be Alice's account ID.
-  - Hint 2: use the estimation RPC method to calculate gas.
-  - Hint 3: `cheque_amount` is unused in this script, so set it to zero.
- 
+  - _Hint 1: This script requires one signer, who has to be Alice's account ID._
+  - _Hint 2: Use the estimation RPC method to calculate gas._
+  - _Hint 3: `cheque_amount` is unused in this script, so set it to zero (it's always safer not to allow scripts to charge you anything if not necessary)._
+    <details>
+    <summary>Click here to unlock the hidden command for the above action.</summary>
+  
+    ```
+    smove node rpc estimate-gas-execute-script -a 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY -s build/car-wash-example/script_transactions/register_new_user.mvt --cheque-limit 0
+    ```
+    </details>
+    
+    ------------------------------------------------------------------------------------------------------------------------------------------------------------
 2. Now, again using Alice's account, buy a coin with a `buy_coin.mv` script:
-  - Hint 1: this script requires one signer which has to be Alice's account ID.
-  - Hint 2: use the estimation RPC method to calculate gas.
-  - Hint 1: because the price of a washing coin is 1 UNIT), we need to write a cheque of around xxxx
- 
-3. Wash the car with Alice's account 
-  - Hint 1: ...
-
-After the car is washed, now check Alice's balance again and see how the above operations changed her balance status:
-![Compare balances after Alice bought a coin from Bob](assets/polkadot.js_buy_coin_balances_after.png)
-
-We hope you finished our tutorial without any issues! And we hope you liked it. If you want to report feedback, please feel free to do so. We are always available at [hello@eiger.co](mailto:hello@eiger.co).
+  - _Hint: Because the price of a washing coin is `1 UNIT`, we need to write a cheque with a value of at least `1000000000000` balances._
```

Comment:
Mhh, what about deleting the last word `balances`? Looks a little strange to me here